### PR TITLE
Readme writing guide, content calendar & avatar update

### PR DESCRIPTION
**Pull request by MitchellShiell**
This update includes the following:

- Tentative Bi-weekly Content Calendar 
- Writing Guide
- Writing templates 
- Updated avatars with Patrick and myself (Mitch). Can update anyone else if missing

Let me know your thoughts; willing to adapt it if need be.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,248 @@
-# ICGC - Software Engineering Blog
+<!--Sources-->
```
Comment (b-f-chan) [inline README.md]: Not asking for changes, but just a question, curious what these listed sources are for?  I am guessing best practice sites/guides you used to revamp the blog?

File: README.md [hunk 2]
```diff
@@ -1,12 +1,248 @@
...
-## Instructions to run a local instance of Jekyll
+<img src="images/OICRHero.jpg">
 
-First clone this repo
+<br />
 
-Then cd into the repo and run:
+<p align="center"><a href="#book-writing-guide">Writing Guide</a>  |  <a href=#computer-running-a-local-instance>Running a Local Instance</a>  |  <a href=#ballot_box-submitting-a-blog-post>Submitting a Blog Post</a>
+
+We are the OICR team of software engineers, infrastructure professionals, and bioinformaticians developing big data tools to help in the fight against cancer. Here we share our successes, failures, curiosities and encouragement for open technical discussion with you, the reader. Welcome!
```
Comment (b-f-chan) [inline README.md]: I always found this wording weird, "...the OICR team of", how about just, "We are OICR's team of"?

File: README.md [hunk 3]
```diff
@@ -1,12 +1,248 @@
...
+
+<br />
+
+# :date: Bi-weekly Content Calendar
```
Comment (b-f-chan) [inline README.md]: Is this calendar public to show people what articles are upcoming?  Or an internal tracker/schedule to keep us on task?  Or both I guess.

Also, should we add the year to the dates?

File: README.md [hunk 4]
```diff
@@ -1,12 +1,248 @@
...
+|May 30|Justin Richardsson|```link google doc```|
+|June 14|Rakesh Mistry|```link google doc```|
+
+<br />
+
+# :book: Writing Guide
+
+**Hello all!**
+
+The following guide is only meant to provide some clarity to how to approach your writing and is by no means prescriptive.
```
Comment (b-f-chan) [inline README.md]: Reword to "...some clarity on how to..."

File: README.md [hunk 5]
```diff
@@ -1,12 +1,248 @@
...
+
+<br />
+
+# :book: Writing Guide
+
+**Hello all!**
+
+The following guide is only meant to provide some clarity to how to approach your writing and is by no means prescriptive. 
+
+In it you will find the information on following:
```
Comment (b-f-chan) [inline README.md]: Reword to "...on the following:"

File: README.md [hunk 6]
```diff
@@ -1,12 +1,248 @@
...
+    - [Conversational Template](#wiki-post)
+
+<br />
+
+<!--
+
+# :runner: :dash: TL;DR
+
+- Word counts can vary depending on the type of article but here's a general outline based on SEO metrics:
+    - Sentences (~ 25 words)
```
Comment (b-f-chan) [inline README.md]: I assume this means 25 words per sentence?

File: README.md [hunk 7]
```diff
@@ -1,12 +1,248 @@
...
+
+ :memo: **Google Docs:** [Wiki Post Template Linked Here](https://docs.google.com/document/d/1BEra-ugD4ETTv_9WQNOwGhMBgry7yeYnW5TEzmDb7Bs/edit?usp=sharing)
+
+ **Titles:** Information on creating a title can be found within the google doc. 
+
+ <!--
+
+|Segments| Description |Approximate Word Count|
+|---|---|---|
+|Introduction|Try not to start off by defining your topic; instead, introduce it your topic by talking about the reader's motivations for reading the post. You can ask yourself "why would someone look for information on this subject, what issues are they trying to solve, what is their unique context? Write down your responses and explain how you plan to resolve them in your article.|1-3 paragraphs|
```
Comment (b-f-chan) [inline README.md]: "...introduce your topic..." (remove `it`)

File: README.md [hunk 8]
```diff
@@ -1,12 +1,248 @@
...
+|Define the topic*|As concisely and definitively as you can, share the definition for that topic.|1-2 paragraphs|
+|Provide context on that topic|You may choose to explore: <ul><li>The history of the topic</li><li>Where the topic fits within a broader field</li><li>How readers might apply it to their own work or life</li><li>Review its effectiveness or compare/contrast it to other related elements)</li></ul>|5 or more paragraphs.|
+|Provide specific examples|Provide one or two case studies to help illustrate your points and provide context (especially if the topic is more abstract). Making it personal can also give the content more authenticity and personality.|5 or more paragraphs|
+|Provide step-by-step advice (optional)|Solve problems in a step by step guide or even link to other resources or posts.|5 or more paragraphs|
```
Comment (b-f-chan) [inline README.md]: Excellent suggestion, however can we or should we add some guidelines on how to properly reference/credit/attribute other sources?

File: README.md [hunk 9]
```diff
@@ -1,12 +1,248 @@
...
+
+-->
+
+## Problem Centered Post
+
+**Description** Present a problem that your reader is experiencing, empathize with that problem, and then solve the problem.
+
+**Approximate word count:** 1500-2500 words. *Tailor as needed.*
+
+:memo: **Google Docs:** [Problem Centered Post Template](https://docs.google.com/document/d/1BStxDjG_iGDc1m-pgOxnMNsh3e5oRmD4llS9DLe92k0/edit?usp=sharing)
```
Comment (b-f-chan) [inline README.md]: The templates are great!  However for each template (problem centred, wiki post, conversational post), can you also link to a good example of what a real one would be like?

File: README.md [hunk 10]
```diff
@@ -122,3 +353,13 @@ https://help.github.com/articles/creating-a-pull-request/
 ### Update blog content (non-posts)
 
 Blog data (projects, team members, positions, ...) are located in `/_data/`, please edit those files in your own branch and submit pull requests towards gh-pages.
+
+<br />
+
+# To Do
```
Comment (b-f-chan) [inline README.md]: Is this To Do list internal or external?  Should probably hide/comment out?

File: _data/authors.yml [hunk 11]
```diff
@@ -84,13 +100,42 @@ linda_xiang:
...
   name: Rakesh Mistry
   position: Software Developer
   info: Full-stack developer with a passion for the front-end and web accessibility.
   avatar: rakesh_mistry.svg
   github: mistryrn
 
+sifa_vahora:
+  name: Sifa Vahora
+  position: Senior Technical Business Analyst
+  info: Always working, 110% is minimum quality requirement. loves food, people and softwares!!! Here to work on a noble cause, using my skills and expertise to make the world a better place!!!
```
Comment (b-f-chan) [inline _data/authors.yml]: Capitalize "Loves food, people..."

File: README.md [hunk 12]
```diff
@@ -1,12 +1,248 @@
...
+|Apr 4|Daniel DeMaria|```link google doc```|
+|Apr 18|Mitchell Shiell|```link google doc```|
+|May 2|Linda Xiang|```link google doc```|
+|May 16|Ummulkiram Rangwala|```link google doc```|
+|May 30|Justin Richardsson|```link google doc```|
+|June 14|Rakesh Mistry|```link google doc```|
+
+<br />
+
+# :book: Writing Guide
```
Comment (joneubank) [inline README.md]: can we move the writing guide to a separate .md file, leaving the main README with the typical content: overview of project, link to live site,  how to interact with the repo (running locally, links to writing guide and topic suggestions, etc.)

File: README.md (hunk 3 above)
Comment (joneubank) [inline README.md]: Let's move the writing schedule out of the readme, its difficult to maintain here and prone to regular changes. Its not what people look for when landing on the repo. As discussed offline, scheduling posts and tracking their content through github issues is a systme that has worked in the past.

**Review by joneubank — CHANGES_REQUESTED** [review]
Great stuff overall. Left a couple comments on how I think it would be better to organize these updates.

File: README.md (hunk 12 above)
Comment (joneubank) [reply README.md]: we can link to it from the main file of course...

File: _data/authors.yml [hunk 13]
```diff
@@ -24,14 +24,21 @@ brandon_chan:
...
   name: Ciarán Schütte
   position: Software Developer
-  info:
+  info: Ciarán, a full stack developer, tried to get an AI to write his bio. He just couldn’t get it to write anything sensible, so instead he chose to write the bio himself using JavaScript and javascript. Especially the bit about delicious garlic.
   avatar: ciaran_schutte.svg
   github: ciaranschutte
 
+dan_demaria:
+  name: Dan DeMaria
+  position: Software Developer III
```
Comment (mcourtot) [inline _data/authors.yml]: suggest removing "III":
- it is inconsistent throughout (some have nothing, some have III, some have 3
- it doesn't mean anything outside of OICR

Comment (MitchellShiell) [reply _data/authors.yml]: Got it!

File: README.md (hunk 12 above)
Comment (MitchellShiell) [reply README.md]: Writing guided moved to a separate file, and topic suggestions were linked on the main readme in the latest commit

File: README.md (hunk 3 above)
Comment (MitchellShiell) [reply README.md]: Done!

File: _data/authors.yml (hunk 13 above)
Comment (joneubank) [reply _data/authors.yml]: In general I would not put levels, just role descriptions. This isn't a registry of our HR job titles.

File: README.md (hunk 1 above)
Comment (MitchellShiell) [reply README.md]: Yup was for my reference, removed them!

File: README.md (hunk 3 above)
Comment (MitchellShiell) [reply README.md]: Deleted calendar, as discussed with Jon we will use the github issues panel for planning and tracking blog post entries

File: README.md (hunk 2 above)
Comment (MitchellShiell) [reply README.md]: You right, the OICR communications doc says something similar to this too good suggestion

File: README.md (hunk 4 above)
Comment (MitchellShiell) [reply README.md]: got it!

File: README.md (hunk 7 above)
Comment (MitchellShiell) [reply README.md]: Table was removed in latest draft

File: README.md (hunk 9 above)
Comment (MitchellShiell) [reply README.md]: Great idea, will have a look around for some examples

File: README.md (hunk 10 above)
Comment (MitchellShiell) [reply README.md]: Removed!

File: _data/authors.yml (hunk 11 above)
Comment (MitchellShiell) [reply _data/authors.yml]: Good find, corrected

File: README.md (hunk 5 above)
Comment (MitchellShiell) [reply README.md]: corrected

File: README.md (hunk 6 above)
Comment (MitchellShiell) [reply README.md]: TLDR was removed on updated draft (slimmed it down for readability)

File: README.md (hunk 8 above)
Comment (MitchellShiell) [reply README.md]: Yes!

File: README.md (hunk 1 above)
Comment (b-f-chan) [reply README.md]: Oh it's OK you could keep them since they were comments, but thanks!

File: README.md (hunk 2 above)
Comment (b-f-chan) [reply README.md]: lgtm thanks!

File: README.md (hunk 3 above)
Comment (b-f-chan) [reply README.md]: Agreed, lgtm thanks!

File: README.md (hunk 4 above)
Comment (b-f-chan) [reply README.md]: change lgtm thanks!

File: README.md (hunk 5 above)
Comment (b-f-chan) [reply README.md]: lgtm thanks!

File: README.md (hunk 6 above)
Comment (b-f-chan) [reply README.md]: change lgtm thanks!

File: README.md (hunk 7 above)
Comment (b-f-chan) [reply README.md]: change lgtm thanks!

File: README.md (hunk 8 above)
Comment (b-f-chan) [reply README.md]: Citation examples look good, thanks!

File: README.md (hunk 9 above)
Comment (b-f-chan) [reply README.md]: Thanks!  And agree this can come in later update, not necessarily this PR

File: README.md (hunk 10 above)
Comment (b-f-chan) [reply README.md]: Change lgtm thanks!

**Review by b-f-chan — APPROVED** [review]
Requested changes done, looks good to me

File: _data/authors.yml [hunk 14]
```diff
@@ -84,13 +100,41 @@ linda_xiang:
   avatar: linda_xiang.svg
   github: lindaxiang
 
+mitchell_shiell:
+  name: Mitchell Shiell
+  position: Outreach Lead
+  info: Scientist, educator, and digital creative using his powers to increase public support and awareness of incredible and innovative technologies. Also an avid hockey player and passionate photographer.
+  avatar: mitchell_shiell.svg
+  github: mitchellshiell
+
```
Comment (Azher2Ali) [inline _data/authors.yml:109]: Azher Ali_Mohammed:
  name: Azher Ali Mohammed
  position: Software Developer
  info: An engineer who loves to solve complex coding and work on logic building. Hold background in Computer Science and Master of Science in Project Management and Analysis.  Also seeks interest in natural and wild life photography.
  avatar: Azher.svg
  github: Azher2Ali

File: writing_guide.md [hunk 15]
```diff
@@ -0,0 +1,120 @@
...
+
+:memo: **Google Docs:** [Problem Centered Post Template](https://docs.google.com/document/d/1BStxDjG_iGDc1m-pgOxnMNsh3e5oRmD4llS9DLe92k0/edit?usp=sharing)
+
+**Titles:** Information on creating a title can be found within the google doc. 
+
+## Wiki Post
+
+**Description:** Gain readers, establish your expertise on a subject, and help many people along the way. Content from wikis or encyclopedias, particularly material that Wikipedia hasn't covered, is also a huge benefit for gaining search traffic. Keep in mind this is still a blog post and not actually a Wikipedia article   
+
+**Approximate word count:** 1500-2500 words. *Tailor as needed.*
```
Comment (joneubank) [inline writing_guide.md:97]: maybe aim for 500-1000?

**Review by joneubank — APPROVED** [review]

**Pull request merged**